tests/unit/tuf_repository_service_worker/test_repository.py

```diff
@@ -27,8 +26,7 @@ def test_refresh_settings_with_none_arg(self):
         test_repo.refresh_settings()
 
         assert (
-            test_repo._worker_settings.to_dict()
-            == repository.worker_settings.to_dict()
+            isinstance(test_repo._worker_settings, repository.Dynaconf) is True
```

Comment:
We cannot check the `test_repo._worker_settings` against anything because the `worker_settings` are created internally in the `MetadataRepository` class instance by calling `get_worker_settings`.